action.yml

```diff
@@ -45,7 +45,7 @@ inputs:
       Name of the package(s) under test, as expected by colcon.
       Passing multiple package names is allowed.
       Package names can be separated by any whitespace character.
-    required: true
+    required: false
```

Comment:
maybe update the description to mention something about everything getting built & tested if the input is left empty?

Also, you might want to add `default: ""` just for consistency